### PR TITLE
Skyve 9 changes

### PR DESCRIPTION
**Pull request by Malcolm-G-Biz**
[Trello Card](https://trello.com/c/XT2M6j0J)

### CODE REVIEW

**Comment by Malcolm-G-Biz** [comment]
[Developer and user guide updates](https://trello.com/c/XT2M6j0J/31-developer-and-user-guide-updates)

File: .project
Comment (benpetito) [inline .project:1]: Do we need this file?

File: _pages/backup-restore.md [hunk 1]
```diff
@@ -25,6 +25,19 @@ To take the backup:
 1. Once the backup is complete, refresh the *Backups* list to check it is complete (it may take some time) - note that the backup process is run as a Skyve *Job* and the progress and completion of the backup can be reviewed from the admin *Jobs* menu
 1. If you wish to download the backup, select the backup from the list and press *Download Backup*
 
+### Backup options
+Before making a backup, there are options found next to the *Backup* button, highlighted below:
+
+- **Content** - Check this to include content in the backup that is made
```
Comment (benpetito) [inline _pages/backup-restore.md]: Maybe mention what content is (files and attachments), and that it's on by default

File: _pages/backup-restore.md [hunk 2]
```diff
@@ -25,6 +25,19 @@ To take the backup:
...
+- **Audit Log** - Check this to include the audit log that are made by the system into the backup created.
```
Comment (benpetito) [inline _pages/backup-restore.md]: Describe what the audit log is (record of changes in the system) and that it's on by default

File: _pages/backup-restore.md [hunk 3]
```diff
@@ -25,6 +25,19 @@ To take the backup:
...
+- **Sensitivity** - While making a backup, skyve has the ability to redact certain data for security purposes. The drop-down is used to select the sensitivity level to be used during redaction. Attributes with security level that is greater than or equal to the security level selected are redacted. Below are the security levels available in ascending order in terms of sensitivity:
```
Comment (benpetito) [inline _pages/backup-restore.md:33]: Isn't it less than or equal?

**Review by benpetito — CHANGES_REQUESTED** [review]

File: _pages/backup-restore.md (hunk 3 above)
Comment (Malcolm-G-Biz) [reply _pages/backup-restore.md:33]: image
I actually got that from the description that is there currently. I've never actually used the feature so I went by that

File: .project
Comment (Malcolm-G-Biz) [reply .project:1]: Added it to gitignore

Comment (Malcolm-G-Biz) [reply .project:1]: It was created by eclipse.

File: _pages/backup-restore.md (hunk 3 above)
Comment (Malcolm-G-Biz) [reply _pages/backup-restore.md:33]: So it should be less than?

**Pull request merged**